models/series/HeatMapSeries.rst

```diff
@@ -28,9 +25,159 @@ To show the x and y values with one digit, use the format string ``"{2:0.0},{4:0
 The default format string for ``HeatMapSeries`` is ``"{0}\n{1}: {2}\n{3}: {4}\n{5}: {6}"``
 
 
-Example
+Example (Linear, Bitmap)
+-------
+
+.. image:: HeatMapSeries_linear.png
+
+.. sourcecode:: csharp
+
+    using OxyPlot;
+    using OxyPlot.Axes;
+    using OxyPlot.Series;
+    
+	namespace ExampleLibrary
+	{
+        
+		public class LinearHeatMapViewModel
+		{
+			private PlotModel modelP1;
+			public LinearHeatMapViewModel()
+			{
+				modelP1 = new PlotModel { Title = "Heatmap" };
+
+				//top axis
+				modelP1.Axes.Add(new LinearColorAxis
+				{
+					Palette = OxyPalettes.Rainbow(100)
+				});
+
+				//generate 1d normal distribution
+				var singleData = new double[100];
+				for(int x = 0; x < 100; ++x)
+					singleData[x] = Math.Exp((-1.0 / 2.0) * Math.Pow(((double)x - 50.0) / 20.0, 2.0));
+				//generate 2d normal distribution
+				var data = new double[100, 100];
+				for(int x = 0; x < 100; ++x)
+					for(int y = 0; y < 100; ++y)
+						data[y, x] = singleData[x] * singleData[(y + 30) % 100] * 100;
+
+				var heatMapSeries = new HeatMapSeries
+				{
+					X0 = 0,
+					X1 = 99,
+					Y0 = 0,
+					Y1 = 99,
+					Interpolate = true,
+					RenderMethod = HeatMapRenderMethod.Bitmap,
+					Data = data
+				};
+
+				modelP1.Series.Add(heatMapSeries);
+                
+			}
+            
+			public PlotModel Model1
+			{
+				get { return modelP1; }
+				set { modelP1 = value; }
+			}
+		}
+	}
+    
+Example (Categorized, Rectangle)
 -------
 
+The following diagram has categories on both, the x-axis and the y-axis.
+
+It visualizes the amount of cake (y-axis) consumed on the specific day of week (x-axis). As can be seen in the diagram, the amount of cake rises from Monday to Sunday.
+
+.. image:: HeatMapSeries_categorized.png
+
 .. sourcecode:: csharp
 
-    var model = new PlotModel { Title = "HeatMapSeries" };
+    using OxyPlot;
+    using OxyPlot.Axes;
+    using OxyPlot.Series;
+    
+	namespace ExampleLibrary
+	{
+        
+		public class LinearHeatMapViewModel
+		{
+			private PlotModel modelP1;
+			public LinearHeatMapViewModel()
+			{
+				modelP1 = new PlotModel { Title = "Cakes per Weekday" };
+
+				//WeekDay-Axis (Bottom)
+				modelP1.Axes.Add(new CategoryAxis
+				{
+					Position = AxisPosition.Bottom,
+					//Key used for specifying this axis in the HeatMapSeries
+					Key = "WeekDayAxis",
+					//Property of the items in ItemSource that should be used as title
+					LabelField = "Title",
+					//Array of Categories (see above), mapped to one of the coordinates of the 2D-data array
+					ItemsSource = new Category[]
```

Comment:
The Category class is not included in the code.
Can a string array be used instead?